### PR TITLE
missing docs entries for remote execution

### PR DESCRIPTION
**Pull request by pondrejk**

### CODE REVIEW

File: docs/api/robottelo.ui.rst [hunk 1]
```diff
@@ -188,6 +188,11 @@
 
 .. automodule:: robottelo.ui.registry
 
+:mod:`tests.foreman.ui.remoteexecution`
+--------------------------------------------
```
Comment (oshtaier) [inline docs/api/robottelo.ui.rst]: too long line

File: docs/api/robottelo.cli.rst [hunk 2]
```diff
@@ -233,4 +238,4 @@ Submodules:
 :mod:`robottelo.cli.usergroup`
 ------------------------------
 
-.. automodule:: robottelo.cli.usergroup
\ No newline at end of file
+.. automodule:: robottelo.cli.usergroup
```
Comment (oshtaier) [inline docs/api/robottelo.cli.rst]: hmm... what is the difference? )

Comment (ldjebran) [inline docs/api/robottelo.cli.rst]: @oshtaier this is a github mystery :)

**Comment by ldjebran** [comment]
Travis is also a reviewer here :)

File: docs/api/robottelo.cli.rst [hunk 3]
```diff
@@ -170,6 +170,11 @@ Submodules:
 
 .. automodule:: robottelo.cli.puppetmodule
 
+:mod:`tests.foreman.cli.remoteexecution`
```
Comment (ldjebran) [inline docs/api/robottelo.cli.rst]: perhaps : tests.foreman.cli.test_remoteexecution

**Comment by pondrejk** [comment]
@ldjebran @oshtaier addressed and added some more missing entries

File: docs/api/robottelo.cli.rst [hunk 4]
```diff
@@ -170,6 +190,11 @@ Submodules:
 
 .. automodule:: robottelo.cli.puppetmodule
 
+:mod:`robottelo.cli.recurring_logic`
+-----------------------------------------
```
Comment (oshtaier) [inline docs/api/robottelo.cli.rst]: strange that travis is not failing on too long lines like this

File: docs/api/robottelo.cli.rst [hunk 5]
```diff
@@ -60,6 +60,11 @@ Submodules:
 
 .. automodule:: robottelo.cli.environment
 
+:mod:`robottelo.cli.erratum`
+--------------------------------
```
Comment (abalakh) [inline docs/api/robottelo.cli.rst]: A nitpick maybe, but above 2 lines should have exactly the same length:
```
:mod:`robottelo.cli.erratum`
----------------------------
```
Same for the rest of lines:
```
:mod:`robottelo.cli.job_invocation`
-----------------------------------
```
and so on.

File: docs/api/robottelo.cli.rst (hunk 2 above)
Comment (abalakh) [inline docs/api/robottelo.cli.rst]: That's really weird behavior. And github's diff contains this change too. @pondrejk could you please double check your commit's diff? If there's nothing changed then maybe better to exclude this change at all? I'm just afraid we can add some invisible unicode character here, as it happened before and caused issues for us.
Or maybe the root cause is different - there's no new line in the end of file. Mind adding it?

**Comment by pondrejk** [comment]
addressed comments, merging since two acks are in place

**Comment by abalakh** [comment]
@pondrejk try to avoid merging own PRs after pushing some changes. Sometimes changes introduce new issues, quick example - #4928. Another final look of one of reviewers may uncover it.
This PR is fine though, as it's simple and only affects docs, but i'd rather not practice such approach at all just in case :)

**Pull request merged**